ibis/sql/postgres/compiler.py

```diff
@@ -584,12 +593,27 @@ def _string_join(t, expr):
 
 def _literal(t, expr):
     dtype = expr.type()
-    value = expr.op().value
+    op = expr.op()
+    value = op.value
 
     if isinstance(dtype, dt.Interval):
         return sa.text("INTERVAL '{} {}'".format(value, dtype.resolution))
     elif isinstance(dtype, dt.Set):
         return list(map(sa.literal, value))
+    # geo spatial data type
+    elif isinstance(expr, ir.PointScalar):
+        result = "POINT({0})".format(_format_point_value(value))
```

Comment:
Remove the `0` from these format strings, it isn't necessary.

Reply:
sure, thanks!